sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/FilterPushdownBenchmark.scala

```diff
@@ -131,211 +132,214 @@ object FilterPushdownBenchmark {
     }
 
     /*
+    OpenJDK 64-Bit Server VM 1.8.0_171-b10 on Linux 4.14.26-46.32.amzn1.x86_64
     Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
     Select 0 string row (value IS NULL):     Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
     ------------------------------------------------------------------------------------------------
-    Parquet Vectorized                            8452 / 8504          1.9         537.3       1.0X
-    Parquet Vectorized (Pushdown)                  274 /  281         57.3          17.4      30.8X
-    Native ORC Vectorized                         8167 / 8185          1.9         519.3       1.0X
-    Native ORC Vectorized (Pushdown)               365 /  379         43.1          23.2      23.1X
+    Parquet Vectorized                            2961 / 3123          5.3         188.3       1.0X
+    Parquet Vectorized (Pushdown)                 3057 / 3121          5.1         194.4       1.0X
```

Comment:
The difference is huge. What happened?

Reply:
yea, I thinks so. But, not sure. I tried to run multiple times though, I didn't get the old performance values...

Reply:
I have not tried it yet, but is it related to the recent change we made in the parquet reader?

Reply:
That might be, but I feel the change was too big... I probably think that I had some mistakes in the last benchmark runs (I've not found why yet though).

Reply:
How about 2.3?

Reply:
Is it a regression?

Reply:
I have time today, so I'll check v2.3.

Reply:
The result in v2.3.1: https://gist.github.com/maropu/88627246b7143ede5ab73c7183ab2128

That is not a regression, but I probably run the bench in wrong branch or commit.
I re-ran the bench in the current master and updated the pr.

how-to-run: I created a new `m4.2xlarge` instance, fetched this pr, rebased to master, and run the bench.

Reply:
Thank you for updating, @maropu .